README.md

```diff
@@ -5,30 +5,30 @@ AWS ECS (Elastic Container Service).
 
 ## Documentation
 
-See https://www.consul.io/docs/ecs for full documentation.
+See https://developer.hashicorp.com/consul/docs/ecs for full documentation.
 
 ## Architecture
 
-![Architecture](https://github.com/hashicorp/terraform-aws-consul-ecs/blob/main/_docs/architecture.png?raw=true)
+![Architecture](./_docs/architecture.png?raw=true)
```

Comment:
This needs to be reverted to the full URL on main or the README won't display correctly on the Terraform rgy.